app.py

```diff
@@ -131,7 +131,7 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        return jsonify(data['skill']), 200
```

Comment:
Since this is a small PR, we can also add unit tests for this functionality here if there is no separate issue for it. What do you think?

Reply:
https://github.com/MLH-Fellowship/orientation-project-python-24.FAL.A/blob/main/test_pytest.py#L59
I believe there are already tests for it. Right?

Reply:
Nice, in that case it's all good 💯